### ISSUE TITLE
Error: Error during loading of extension ChainRulesCoreExt of LogExpFunctions, use `Base.retry_load_extensions()` to retry.

### ISSUE THREAD

**Issue by lmiq** [JuliaStats/KernelDensity.jl#112]
When trying to install `KernelDensity`, I'm getting:

```julia
julia> using KernelDensity
 │ Package KernelDensity not found, but a package named KernelDensity is available from a registry. 
 │ Install package?
 │   (ribonuclease) pkg> add KernelDensity 
 └ (y/n/o) [y]: y
   Resolving package versions...
    Updating `~/Documents/rafael/ribonuclease/Project.toml`
  [5ab0869b] + KernelDensity v0.6.5
    Updating `~/Documents/rafael/ribonuclease/Manifest.toml`
  [621f4979] + AbstractFFTs v1.2.1
  [79e6a3ab] + Adapt v3.5.0
  [13072b0f] + AxisAlgorithms v1.0.1
  [49dc2e85] + Calculus v0.5.1
  [d360d2e6] + ChainRulesCore v1.15.7
  [b429d917] + DensityInterface v0.4.0
  [31c24e10] + Distributions v0.25.83
  [fa6b7ba4] + DualNumbers v0.6.8
  [7a1cc6ca] + FFTW v1.5.0
  [1a297f60] + FillArrays v0.13.7
  [34004b35] + HypergeometricFunctions v0.3.11
  [a98d9a8b] + Interpolations v0.14.7
  [3587e190] + InverseFunctions v0.1.8
  [5ab0869b] + KernelDensity v0.6.5
  [6fe1bfb0] + OffsetArrays v1.12.9
  [90014a1f] + PDMats v0.11.16
  [1fd47b50] + QuadGK v2.8.1
  [c84ed2f1] + Ratios v0.4.3
  [79098fc4] + Rmath v0.7.1
  [4c63d2b9] + StatsFuns v1.2.1
  [efce3f68] + WoodburyMatrices v0.5.5
  [f5851436] + FFTW_jll v3.3.10+0
  [1d5cc7b8] + IntelOpenMP_jll v2018.0.3+2
  [856f044c] + MKL_jll v2022.2.0+0
  [f50d1b31] + Rmath_jll v0.4.0+0
  [4af54fe1] + LazyArtifacts
  [1a1011a3] + SharedArrays
  [4607b0f0] + SuiteSparse
Precompiling environment...
  2 dependencies successfully precompiled in 3 seconds. 179 already precompiled.
  1 dependency had warnings during precompilation:
┌ KernelDensity [5ab0869b-81aa-558d-bb23-cbf5423bbe9b]
│  ┌ Error: Error during loading of extension ChainRulesCoreExt of LogExpFunctions, use `Base.retry_load_extensions()` to retry.
│  └ @ Base loading.jl:1191
│  ┌ Error: Error during loading of extension InverseFunctionsExt of LogExpFunctions, use `Base.retry_load_extensions()` to retry.
│  └ @ Base loading.jl:1191
└  
┌ Error: Error during loading of extension ChainRulesCoreExt of LogExpFunctions, use `Base.retry_load_extensions()` to retry.
└ @ Base loading.jl:1191
┌ Error: Error during loading of extension InverseFunctionsExt of LogExpFunctions, use `Base.retry_load_extensions()` to retry.
└ @ Base loading.jl:1191

julia> Base.retry_load_extensions()
┌ Error: Error during loading of extension ChainRulesCoreExt of LogExpFunctions, use `Base.retry_load_extensions()` to retry.
└ @ Base loading.jl:1191
┌ Error: Error during loading of extension InverseFunctionsExt of LogExpFunctions, use `Base.retry_load_extensions()` to retry.
└ @ Base loading.jl:1191
```

The error appears as well in subsequent `using`s.

```julia
julia> versioninfo()
Julia Version 1.9.0-beta4
Commit b75ddb787ff (2023-02-07 21:53 UTC)
Platform Info:
  OS: Linux (x86_64-linux-gnu)
  CPU: 12 × 11th Gen Intel(R) Core(TM) i5-11500H @ 2.90GHz
  WORD_SIZE: 64
  LIBM: libopenlibm
  LLVM: libLLVM-14.0.6 (ORCJIT, tigerlake)
  Threads: 12 on 12 virtual cores

(ribonuclease) pkg> st
Status `~/Documents/rafael/ribonuclease/Project.toml`
  [46823bd8] Chemfiles v0.10.3
  [6f35c628] ComplexMixtures v0.6.2
  [5ab0869b] KernelDensity v0.6.5
  [e29189f1] PDBTools v0.13.7
  [91a5bcdd] Plots v1.38.5

```

**Comment by ararslan** [comment]
Can you try again with 1.9.0-rc1?

**Comment by lmiq** [comment]
It's solved in 1.9.0+rc1. Thanks.
